sycl/include/CL/sycl/handler.hpp

```diff
@@ -89,6 +89,19 @@ decltype(member_ptr_helper(&F::operator())) argument_helper(F);
 
 template <typename T>
 using lambda_arg_type = decltype(argument_helper(std::declval<T>()));
+
+// Helper struct to get a kernel name type based on given \c Name and \c Type
```

Comment:
Are Doxygen markings applied to regular comment blocks?
Shouldn't they have to be "special comment blocks"?
http://www.doxygen.nl/manual/docblocks.html#cppblock

Reply:
OK